runestone/selectquestion/js/selectone.js

```diff
@@ -6,8 +6,8 @@
 import {
     renderRunestoneComponent,
     createTimedComponent,
-} from "../../common/js/renderComponent";
-import RunestoneBase from "../../common/js/runestonebase";
+} from "../../common/js/renderComponent.js";
+import RunestoneBase from "../../common/js/runestonebase.js";
```

Comment:
As far as I understand it there is no difference between including and not including the .js ??

But we should probably be consistent to reduce possible confusion.

Reply:
Based on https://webpack.js.org/api/module-methods/, I started using `.js` as the standard extension. (The code still works without it, but I think it's best to follow this recommendation.)